nupic/algorithms/anomaly_likelihood.py

```diff
@@ -106,11 +112,7 @@ def __init__(self,
     self._probationaryPeriod = claLearningPeriod + estimationSamples
     self._claLearningPeriod = claLearningPeriod
 
-    # How often we re-estimate the Gaussian distribution. The ideal is to
-    # re-estimate every iteration but this is a performance hit. In general the
-    # system is not very sensitive to this number as long as it is small
-    # relative to the total number of records processed.
-    self._reestimationPeriod = 100
+    self._reestimationPeriod = reestimationPeriod
```

Comment:
@scottpurdy: a couple of comments regarding `__setstate__`:

1. The same member variable `_reestimationPeriod` existed before; if someone saved it having the value of 100, then it would have been acceptable to them, and there is no need to fix it up upon de-serialization.
2. Also, `__setstate__` doesn't exist for this class, anyway. But I think that the above item is what really matters.